### DISCUSSION TITLE
Create FAQ

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#88] [Ideas]
Greetings. I find it interesting to create a FAQ.md file to answer the most frequent questions when working with the platform. This benefits users new to the platform. It also avoids duplicates within the issue or discussions.

There are several unknown questions that need to be clarified such as:

* What is FTA (free-to-air) or Freeview? Do you consider it a requirement to add to this repository? (maybe yes, although for some it is geoblocked for the public outside the country of origin)
* If the FTA channel broadcasts only sporting events or pay programming, will it be added to the list? (the answer is no because of the blocklist issue)
* Why is recommended to end the static "playlist.m3u" or "index.m3u" at the end of the url? (the answer is related to adaptive stream, insead of tokens or chunks files what usually expire quickly)
* Why don't we accept XStreamCode in streams? (answer, because this a safe)
* Is it possible to add streams from Youtube, Dailymotion or Twitch? (the answer is yes, when it is not geolocalized and the initial url is fullspeed.tv)
* Is it possible to add audio-only streams, like FM radio? (no)
* Why are there some ids with call sign (WATB-TV.us) and others with alphanumeric id (Mychannel.us)? (perhaps due of how the broadcasting behave in DTTV, analog TV or web stream)
* There are source for verify call sign and coverage area for some TV stations? [FCCData](https://fccdata.org/) for US, Canada, Mexico UK, Austrailia and Japan.
* Why some stream have empty id? (because their entries are not registered in the database)
* What is a "daily update" and what is the benefit of this check? (A "daily update" is a frequent revision that ensures which streams are working, as well as showing technical details in the API)
* Is it possible to view channels directly from the site? (no, because this option generates confusion by offering the playlists only for public access with the entire list of channels accessible on other websites)
* What happens when a channel is labeled NSFW and does NSFW apply if there is adult programming? 
* Do I have to pay to use this list? (no)
* Are you planning to include a VOD of the programs broadcast on this playlist? (no)
* And other more.

**Comment by BellezaEmporium** [comment]
Agree. As time moves on, more and more users are contributing. A good Q&A is primordial for clearing out certain issues.

**Reply by sguinetti** [reply]
I would create in next PR.

**Comment by sguinetti** [comment]
Created #7954.
